app/src/main/java/com/example/androidacademyhw/details/FragmentMoviesDetails.kt

```diff
@@ -52,17 +57,33 @@ class FragmentMoviesDetails : Fragment()  {
         val movie = arguments?.getSerializable(Tags.KEY_MOVIE) as Movie
 
         with(binding) {
-            age.text = getString(R.string.details_item_text_pg, movie.pg)
-            movieNameText.text = movie.name
-            genre.text = movie.tags
-            ratingbar.rating = movie.rating.toFloat()
-            reviewAmount.text = getString(R.string.details_text_reviews, movie.reviewCount)
-            movieDescription.text = movie.storyLine
+
+            Glide.with(root.context)
+                .load(movie.poster)
+                .into(movieLogoImage)
+
+            age.text = getString(R.string.details_item_text_pg, movie.minimumAge)
+            movieNameText.text = movie.title
+            genre.text = movie.genres.map{it.name}.joinToString()
```

Comment:
press alt+l on the whole file to reformat the code

Reply:
+